pkg/controller/controller.go

```diff
@@ -91,13 +94,26 @@ func New(cfg Config, smclient *monitoringv1alpha1.MonitoringV1alpha1Client) *Con
 		},
 	})
 
+	controllerNamespace := os.Getenv("KUBELESS_NAMESPACE")
+	kubelessConfig := os.Getenv("KUBELESS_CONFIG")
```

Comment:
we better set here the default values if these don't exists (just like in the CLI)

Reply:
Done.